src/libraries/System.Linq/src/System/Linq/Last.cs

```diff
@@ -30,11 +30,24 @@ public static TSource Last<TSource>(this IEnumerable<TSource> source, Func<TSour
             return last!;
         }
 
-        public static TSource? LastOrDefault<TSource>(this IEnumerable<TSource> source) =>
-            source.TryGetLast(out bool _);
+        public static TSource? LastOrDefault<TSource>(this IEnumerable<TSource> source)
+            => source.TryGetLast(out _);
```

Comment:
This file is using => on the next line, whereas First.cs is using it on the previous line.

Reply:
I tried to preserve the existing style in each file. Is there one that's preferred? I personally try to do things like the example in Last.cs.

Reply:
We don't have a defined style.  We probably should.  Personally I always put them at the end of the previous line, but I know some other folks prefer it the other way.

If the file was already doing it this way, ok.

Reply:
I checked the originals and they're putting the lambda on the same line as the declaration. I'll return it to that style.